backend/app/processing/data_loader.py

```
@@ -161,6 +161,26 @@ def load_excel(file_path: str) -> List[str]:
         return []
 
 
+def split_text_to_chunks(text: str, max_length: int = 1000) -> List[str]:
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)